packages/boilerplate-generator/package.js

```diff
@@ -22,3 +25,16 @@ Package.onUse(function (api) {
     'boilerplate_web.cordova.html'
   ], 'server');
 });
+} else {
+
+Package.onUse(function (api) {
+  api.use([
+    'underscore',
```

Comment:
Given that we're relying on underscore anyway, maybe just use [`_.template`](http://underscorejs.org/#template) with a template stored in a file and read by Assets.getText?